libs/langchain-community/src/vectorstores/neo4j_vector.ts

```diff
@@ -577,13 +577,18 @@ export class Neo4jVectorStore extends VectorStore {
     );
   }
 
-  async similaritySearch(query: string, k = 4): Promise<Document[]> {
+  async similaritySearch(
+    query: string,
+    k = 4,
+    params: Record<string, any> = {}
```

Comment:
The convention has been to treat this final parameter as a search filter - would be nicer if possible to support adding `invocationParams` when initializing the client and spreading them when running the query.

Reply:
What are some params you'd want to pass here?

Reply:
you could personalize responses based on user ids, user role, locations, and probably more. This only works when you provide a corresponding retrieval_query with relevant parameters.

Reply:
Let's roll with it then